index.html

```diff
@@ -29,7 +29,7 @@ <h4 class="border-bottom py-3 textSpacing_04">
             as your appetite guides you to consume according to your body's
             needs.
           </p>
-          <h6 class="textRed textBold">BE MINDFUL</h6>
+          <h6 class="textRed textBold text-uppercase">BE MINDFUL</h6>
```

Comment:
Better to scan through your custom class name again. You can use `text-danger` and `font-weight-bold` from Bootstrap.